src/JWTGuard.php

```diff
@@ -121,6 +135,10 @@ public function attempt(array $credentials = [], bool $login = true)
         $this->lastAttempted = $user = $this->provider->retrieveByCredentials($credentials);
 
         if ($this->hasValidCredentials($user, $credentials)) {
+            $this->events->dispatch(
+                new JWTAttempt($user)
+            );
```

Comment:
I think this should be dispatched regardless of valid credentials

Reply:
I agree, ready!